updates.jenkins.io.tf

```
@@ -0,0 +1,56 @@
+# Storage account
+resource "azurerm_resource_group" "updates_jenkins_io" {
+  name     = "updates-jenkins-io"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "azurerm_storage_account" "updates_jenkins_io_files" {
+  name                          = "updatesjenkinsiofiles"
```

```suggestion
  name                          = "updatesjenkinsio"
```

=> A storage account can have different services: files, blob storages, etc.